StoreManagementService/src/PBJ.StoreManagementService.Business/Services/UserService.cs

```diff
@@ -0,0 +1,143 @@
+﻿using AutoMapper;
+using PBJ.StoreManagementService.Business.Constants;
+using PBJ.StoreManagementService.Business.Dtos;
+using PBJ.StoreManagementService.Business.Exceptions;
+using PBJ.StoreManagementService.Business.Services.Abstract;
+using PBJ.StoreManagementService.DataAccess.Entities;
+using PBJ.StoreManagementService.DataAccess.Repositories.Abstract;
+
+namespace PBJ.StoreManagementService.Business.Services
+{
+    public class UserService : IUserService
+    {
+        private readonly IUserRepository _userRepository;
+        private readonly IPostRepository _postRepository;
+        private readonly ICommentRepository _commentRepository;
+        private readonly IMapper _mapper;
+
+        public UserService(IUserRepository userRepository,
+            IPostRepository postRepository,
+            ICommentRepository commentRepository,
+            IMapper mapper)
+        {
+            _userRepository = userRepository;
+            _postRepository = postRepository;
+            _commentRepository = commentRepository;
+            _mapper = mapper;
+        }
+
+        public async Task<List<UserDto>> GetAmountAsync(int amount)
+        {
+            var users = await _userRepository.GetAmountAsync(amount);
+
+            if (users.Count == 0)
+            {
+                throw new NotFoundException(ExceptionMessages.USERS_EMPTY_MESSAGE);
+            }
+
+            return _mapper.Map<List<UserDto>>(users);
+        }
+
+        public async Task<List<UserDto>> GetFollowersAsync(int userId, int amount)
+        {
+            var followers = await _userRepository.GetFollowersAsync(userId, amount);
+
+            if (followers.Count == 0)
```

Comment:
Why are you check count of followers and so on?
Just return empty list

Reply:
Don't understand what your mean "Just return empty list"

Reply:
Just return followers without checking, even if the collection is empty. It seems like it is a possible case that a user not exists followers.